integration/revoke-document.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Selector } from "testcafe";
+import { enterPassword, loadConfigFile } from "./helper";
+import { homedir } from "os";
+import { join } from "path";
+import { existsSync, readFileSync } from "fs";
+import { file } from "jszip";
+
+fixture("Revoke flow").page`http://localhost:3000`;
+
+const Config = "./../src/test/fixtures/sample-config-local.json";
+const Title = Selector("h1");
+const Button = Selector("button");
+const ProgressBar = Selector("[data-testid='progress-bar']");
+const SubmitButton = Selector("[data-testid='form-submit-button']");
+const DownloadLink = Selector("[data-testid='download-file-button']");
+
+function getFileDownloadPath(fileName: string): string {
+  return join(homedir(), "Downloads", fileName);
+}
+
+// From https://stackoverflow.com/a/57624660/950462
+const waitForFileDownload = async (t: TestController, filePath: string): Promise<boolean> => {
+  // Timeout after 10 seconds
+  for (let i = 0; i < 100; i++) {
+    if (existsSync(filePath)) return true;
+    await t.wait(100);
+  }
+  return existsSync(filePath);
+};
+
+test("should revoke a document on local blockchain correctly", async (t) => {
+  // Upload config file
```

Comment:
can we prepare a local bc fixture before hand, then revoke that file directly? keep this test small

Reply:
erm i not sure if thats doable cos: 
1) every time we start an instance of the blockchain and run integration the document store is different no?
2) if the document store is the same, is it possible to issue a document pre-load of the blockchain?

Reply:
1. no. locally, it can be deterministic
2. yeah can

refer to tt-web implementation for inspiration. then again i think this repo needs refactoring, pr might get big + not relating to intent. let's leave it for now, but we should clean it up someday